modules/core/src/main/scala/org/tesselation/http/routes/RegistrationRoutes.scala

```diff
@@ -0,0 +1,27 @@
+package org.tesselation.http.routes
+
+import cats.effect.Async
+
+import org.tesselation.domain.cluster.Cluster
+
+import org.http4s.HttpRoutes
+import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+
+final case class RegistrationRoutes[F[_]: Async](cluster: Cluster[F]) extends Http4sDsl[F] {
+
+  implicit val logger = Slf4jLogger.getLogger[F]
+
+  private[routes] val prefixPath = "/registration"
+
+  private val httpRoutes: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "request" =>
+      Ok(cluster.getRegistrationRequest)
```

Comment:
Is this a debug-only endpoint? Should it be committed?

Reply:
No, debug endpoints are in `DebugRoutes`. Here is the proper endpoint for asking for a registration request.